src/errors/error-handler.js

```diff
@@ -0,0 +1,20 @@
+import VASTParserError from './vast-parser-error'
+
+export default class ErrorHandler {
+  constructor (strict) {
+    this._strictMode = strict
+  }
+
+  fail (error, errorCode = 101) {
```

Comment:
It seems a bit random to let the error handler throw schema validation errors by default.
- if we really want that, lets put that in a constant to make it more clear what the default error code is and what it means
- otherwise, lets remove the default parameter in favor of setting the error code explicitly everywhere

WDYT?